test/scheduler_patch_test.rb

```diff
@@ -0,0 +1,49 @@
+# vim:fileencoding=utf-8
+require_relative 'test_helper'
+require 'resque/scheduler_patch'
+
+context 'Resque::SchedulerPatch' do
+  setup do
+    Resque::Scheduler.clear_schedule!
+    Resque::Scheduler.configure do |c|
+      c.dynamic = false
+      c.quiet = true
+      c.poll_sleep_amount = nil
+    end
+
+    @scheduler = Resque::Scheduler.dup
+    @scheduler.singleton_class.prepend(Resque::SchedulerPatch)
+  end
+
+  teardown do
+    Resque.redis.redis.flushall
+  end
+
+  test 'can enqueue into new queue format' do
+    t = Time.now + 60 # in the future
+    Resque.enqueue_at(t, SomeIvarJob)
+    Resque.enqueue_at(t, SomeIvarJob)
+    @scheduler.handle_delayed_items(t)
+
+    assert_equal 0, Resque.delayed_queue_schedule_size
```

Comment:
You never asserted that the queue grew at all. `enqueue_at` could be a no-op and this would pass

Reply:
I guess I assumed the other tests give us hat safety but I'll add a check anyways